src/core/commands/command.js

```diff
@@ -19,7 +20,11 @@ class Command {
     if (err instanceof TeamNotConfiguredError) {
       bot.reply(userMessage, 'Logz.io integration is not configured!\n' +
         'Use `setup` command to configure the Logz.io integration and try again.');
-    } else {
+    }
+    if (err instanceof RateLimitExceededError) {
```

Comment:
replace with `else if`

Reply:
why? is it possible that err will be instance of TeamNotConfiguredError and also instance of RateLimitExceededError?